ena_upload/templates/ENA_template_runs.xml

```diff
@@ -1,12 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?python
+import pandas as pd
+def attributetest(row, column):
+    if hasattr(row, column) and pd.notna(row[column]) and not row[column].isspace() and str(row[column]).lower() not in ['nan', 'na']:
+        return True 
+    else:
+        return False
+?>
 <RUN_SET xmlns:py="http://genshi.edgewall.org/"
     xmlns:xi="http://www.w3.org/2001/XInclude"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_6/SRA.run.xsd">
     <py:for each="alias, experiment_alias in run_groups.iteritems()">
     <RUN alias="${alias}" center_name="${center}">
```

Comment:
While not clear where to add it in the webin submissions portal, a `run_center` and `run_date` can be added to the run.xml as mentioned in the support guide [here](https://ena-docs.readthedocs.io/en/latest/exp_01.html?highlight=run_center%20run_date#rna-seq-run-attributes-to-include-in-your-run-objects)

It is not clear why it is only mentioned in conjunction with RNA-seq objects in the support guide, as no such restriction is mentioned in the SRA.run.xsd

Reply:
these are attributes and are on the same level as `alias` and `center_name` if I read the XML correctly. Sadly I cant put optional fields as attributes at the moment